maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,86 @@
+# ${artifactId}
+
+#[[
+## Designing
+]]#
+While designing your service it is useful to read [designing services](https://developer.lightbend.com/docs/akka-serverless/designing/index.html)
+
+#[[
+## Developing
+]]#
+This project has a bare-bones skeleton service ready to go, but in order to adapt and
+extend it it may be useful to read up on [developing services](https://developer.lightbend.com/docs/akka-serverless/developing/index.html)
+and in particular the [Java section](https://developer.lightbend.com/docs/akka-serverless/java-services/index.html)
+
+#[[
+## Building
+]]#
+To build, at a minimum you need to generate and process sources, particularly when using an IDE.
+A convenience is compile your project:
```

Comment:
Not sure what this means...  How about just
"To compile your project:"

(Ditto in maven-java/akkaserverless-maven-archetype-value-entity/src/main/resources/archetype-resources/README.md)

Reply:
```suggestion
To compile your project:
```